src/Sil/Component/Address/Model/Address.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licenced under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Sil\Component\Address\Model;
+
+class Address implements AddressInterface
+{
+    /**
+     * The street name, number, district, etc.
+     *
+     * @var string
+     */
+    protected $street;
+
+    /**
+     * Any other information that is usefull for the address.
+     *
+     * @var string
+     */
+    protected $other;
+
+    /**
+     * City.
+     *
+     * @var CityInterface
+     */
+    protected $city;
+
+    public function __construct(string $street, CityInterface $city)
+    {
+        $this->street = $street;
+        $this->city = $city;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStreet(): string
+    {
+        return $this->street;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getOther(): ?string
+    {
+        return $this->other;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOther(string $other): void
```

Comment:
should be `?string $other` if getOther() can return null.

Reply:
Hum, setting a null value isn't very logical... The interface tell that the value should not be initialized when creating object.
Maybe adding a `clearOther(): void` method to clear the value would be more semantic ?